platform/core/src/capabilities/InstantiationCapability.js

```diff
@@ -45,19 +48,26 @@ define(
          * have been persisted, nor will it have been added to the
          * composition of the object which exposed this capability.
          *
+         * @param {object} the model for the new domain object
          * @returns {DomainObject} the new domain object
          */
         InstantiationCapability.prototype.instantiate = function (model) {
+            var parsedId =
+                    this.identifierService.parse(this.domainObject.getId()),
+                space = parsedId.getDefinedSpace(),
```

Comment:
Hrm, I'm wondering how we'll actually handle default spaces in the current VISTA plan.  For example, an admin could have access multiple persistence spaces, but what about another user whose permissions only allow access to a subset of those spaces?  A single default value for space isn't guaranteed to resolve.  Having no default space would also cause issues.

I think it's safest to require each persistence provider to specify at least one space, and each space must define it's own root, but then there's the default root, which is added via an extension category and can't be easily removed.  Could override the RootModelProvider but that seems messy.

Thoughts?